explainer.md

```diff
@@ -1,751 +1,423 @@
-
-https://user-images.githubusercontent.com/93594/141118353-d62d19a1-0964-4fa0-880f-bdde656ce899.mp4
-
-The element is captured without the effects (such as opacity and filters) from
-parent elements. Effects on the element itself are baked into the image.
-However, the element is captured without transforms, as those transforms are
-reapplied later. The root is always captured as a single image, with the other
-transition elements removed (similar to how compositing works today).
-
-Capturing an element in this way isn't a new concept to the platform, as
-[`element()`](<https://developer.mozilla.org/en-US/docs/Web/CSS/element()>) in
-CSS performs a similar action. The differences are documented later.
-
-Capturing as a CSS image avoids the interactivity risks, complexities, and
-memory impact of fully preserving these parts of Page-A as live DOM. On the
-other hand, it means that the capture will be 'static'. If it includes things
-like gifs, video, or other animating content, they'll be frozen on the frame
-they were displaying when captured.
-
-#### Image Size
-
-The size of the image cached for an element is equal to the element's [ink
-overflow
-rectangle](https://drafts.csswg.org/css-overflow-3/#ink-overflow-rectangle).
-This allows exposing parts of an element during the transition which may have
-been hidden earlier. The user-agent is allowed to clip the image to an
-implementation defined size (a common case would be the max texture size
-supported by the device). When caching a subset of the element due to this
-constraint, the area within the element cached by the user-agent is the area
-closest to the viewport.
+https://user-images.githubusercontent.com/93594/184085118-65b33a92-272a-49f4-b3d6-50a8b8313567.mp4
 
-The size of the root image and the area captured follows a pattern similar to
-shared elements. However, since the root image is generated using the root
-stacking context it is likely to be clipped to an implementation defined size in
-most cases.
-
-An alternate choice was to clip the element to viewport bounds to limit memory
-use, particularly for the root element. This can be added as a perfomance hint
-from the developer in future iterations. See issues
-[72](https://github.com/WICG/shared-element-transitions/issues/72) and
-[73](https://github.com/WICG/shared-element-transitions/issues/73) for detailed
-discussion on this topic.
-
-The single image mode works great for the share button and the root, as their
-transitions can be represented by simple transforms. However, the header changes
-size without stretching its shadow, and the content of the header moves
-independently and doesn't stretch. There's another mode for that:
+Page transitions not only look great, they also communicate direction of flow, and make it clear which elements are related from page to page. They can even happen during data fetching, leading to a faster perception of performance.
```

Comment:
I removed it because the "flash of white" is wrong, and it seemed to focus on MPA. But yeah, cutting it has lost too much. I'll add some of this back.